sklearn/tests/test_common.py

```diff
@@ -307,3 +308,37 @@ def test_search_cv(estimator, check, request):
 def test_check_n_features_in_after_fitting(estimator):
     _set_checking_parameters(estimator)
     check_n_features_in_after_fitting(estimator.__class__.__name__, estimator)
+
+
+# TODO: When more modules get added, we can remove it from this list to make
+# sure it gets tested. After we finish each module we can move the checks
+# into check_estimator.
+# NOTE: Metaestimators that delegates validation to the inner estimator is
+# is actually checking that the inner estimator checks for column name
```

Comment:
I think "is" is duplicated but I have a hard time parsing the sentence either way.